src/Auditable.php

```diff
@@ -282,18 +283,24 @@ public function toAudit(): array
 
         $tags = implode(',', $this->generateTags());
 
-        return $this->transformAudit([
+        $data = [
             'old_values'     => $old,
             'new_values'     => $new,
             'event'          => $this->auditEvent,
             'auditable_id'   => $this->getKey(),
             'auditable_type' => $this->getMorphClass(),
-            $userForeignKey  => $this->resolveUser(),
+            $userForeignKey  => $this->resolveUserId(),
```

Comment:
Then here, we replace
`$this->resolveUserId()`

with

`$user->getAuthIdentifier()`